dali/base/dautils.cpp

```diff
@@ -47,6 +47,59 @@
 #define MIN_REDIRECTION_LOAD_INTERVAL 1000
 
 
+bool isHostInPlane(IPropertyTree *plane, const char *host, bool ipMatch)
+{
```

Comment:
I am confused with ipMatch parameter. It seems always true. Why is it needed?

Reply:
it's debatable whether you want to resolve all host names to see if a match, or if in other situations it should be a text only match, e.g. resolving to IP's involves a DNS lookup and might fail/stall.
The option is there for the alternative.

Reply:
Ok